### PR TITLE
🐛Source Klaviyo: add missing field

### PR DESCRIPTION
**Pull request by askarpets**
## What
Add field `archived` to `flows` stream.

## How
Add `"archived": { "type": "boolean" },` to `flows.json` and update acceptance tests expected records.

## Recommended reading order
1. `flows.json`
2. `expected_records.jsonl`
3. Unit tests

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by airbyte-oss-build-runner** [comment]
##  source-klaviyo test report (commit [`fd62eb6a8c`](https://github.com/airbytehq/airbyte/pull/31562/commits/fd62eb6a8c7b4be9498a7c1ecbd9ede43ce299f5)) - ❌

⏲️  Total pipeline duration: 02mn58s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-klaviyo docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Acceptance tests                                               | ❌       |
| Code format checks                                             | ✅       |
| Validate metadata for source-klaviyo                           | ✅       |
| Connector version semver check                                 | ✅       |
| Connector version increment check                              | ✅       |
| QA checks                                                      | ❌       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/source-klaviyo-add-missing-field/1697639291/fd62eb6a8c7b4be9498a7c1ecbd9ede43ce299f5/source-klaviyo/0.4.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:fd62eb6a8c7b4be9498a7c1ecbd9ede43ce299f5)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-klaviyo test
```

**Review by bazarnov — APPROVED** [review]

File: airbyte-integrations/connectors/source-klaviyo/unit_tests/test_source.py [hunk 1]
```diff
@@ -35,3 +36,11 @@ def test_check_connection(requests_mock, status_code, response, is_connection_su
     success, error = source.check_connection(logger=None, config={"api_key": "api_key"})
     assert success is is_connection_successful
     assert error == error_msg
+
+
+def test_streams():
+    source = SourceKlaviyo()
+    config = {"api_key": "some_key", "start_date": pendulum.datetime(2020, 10, 10).isoformat()}
+    streams = source.streams(config)
+    expected_streams_number = 8
```
Comment (erohmensing) [inline airbyte-integrations/connectors/source-klaviyo/unit_tests/test_source.py:45]: for good measure can we add an assertion that all of the stream names are unique?

**Review by erohmensing — APPROVED** [review]
Please update the changelog in the connector docs.

Otherwise, fixing a missed field, adding tests, populating a test stream? Great, no notes 👌🏻

File: airbyte-integrations/connectors/source-klaviyo/unit_tests/test_source.py (hunk 1 above)
Comment (askarpets) [reply airbyte-integrations/connectors/source-klaviyo/unit_tests/test_source.py:45]: Updated, thanks for the advice!

**Comment by airbyte-oss-build-runner** [comment]
##  source-klaviyo test report (commit [`907cfda43e`](https://github.com/airbytehq/airbyte/pull/31562/commits/907cfda43ed9fc09a13e9bd25759059b4318b75c)) - ✅

⏲️  Total pipeline duration: 03mn22s 

| Step                                                           | Result   |
|:---------------------------------------------------------------|:---------|
| Build source-klaviyo docker image for platform(s) linux/x86_64 | ✅       |
| Unit tests                                                     | ✅       |
| Acceptance tests                                               | ✅       |
| Code format checks                                             | ✅       |
| Validate metadata for source-klaviyo                           | ✅       |
| Connector version semver check                                 | ✅       |
| Connector version increment check                              | ✅       |
| QA checks                                                      | ✅       |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/source-klaviyo-add-missing-field/1697699117/907cfda43ed9fc09a13e9bd25759059b4318b75c/source-klaviyo/0.4.0/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:907cfda43ed9fc09a13e9bd25759059b4318b75c)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-klaviyo test
```

**Pull request merged**